backend/.env.example

```
@@ -22,6 +22,12 @@ ACCESS_TOKEN_EXPIRE_MINUTES=10080
 EXECUTOR_CANCEL_TASK_URL=http://localhost:8001/executor-manager/tasks/cancel
 EXECUTOR_DELETE_TASK_URL=http://localhost:8001/executor-manager/executor/delete
 
+# Wework direct chat configuration
+# Allowed origins are sent to local Executor during device registration.
+WEWORK_DIRECT_CHAT_ALLOWED_ORIGINS=["http://127.0.0.1:1420","http://localhost:1420","tauri://127.0.0.1","tauri://localhost"]
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)